internal/pkg/store/sql/sqlite/cgo_free_sqllite.go

```diff
@@ -0,0 +1,85 @@
+//go:build !cgo
+
+/*
+ * Copyright 2023 EMQ Technologies Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sqlite
+
+import (
+	"database/sql"
+	"fmt"
+	"os"
+	"path"
+	"sync"
+
+	"github.com/lf-edge/ekuiper/internal/conf"
+	"github.com/lf-edge/ekuiper/internal/pkg/store/definition"
+
+	// introduce sqlite
+	_ "github.com/logoove/sqlite"
```

Comment:
Why using this?

Reply:
`czniz/sqlite` use `sqlite` as driver name, which is different from the original sqlite driver and will cause problems if we use `dburl` to parse url. This one forked the `czniz/sqlite` and fixed this problem.